src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisCacheWrapper.cs

```diff
@@ -8,12 +8,16 @@ namespace OrchardCore.Redis.Services;
 /// <summary>
 /// Wrapper preventing the <see cref="RedisCache"/> to dispose a shared <see cref="IConnectionMultiplexer"/>.
 /// </summary>
-public class RedisCacheWrapper : IDistributedCache
+public class RedisCacheWrapper : IDistributedCache, IDisposable
 {
     private readonly RedisCache _cache;
 
     public RedisCacheWrapper(IOptions<RedisCacheOptions> optionsAccessor) => _cache = new RedisCache(optionsAccessor);
 
+#pragma warning disable CA1816
```

Comment:
This warning should not be disabled, either call `GC.SuppressFinalize` or seal the class (which I think is preferred here).

Reply:
I was thinking of implementing a disposing pattern, but seal the class is fine